codalab/lib/metadata_util.py

```diff
@@ -60,7 +60,7 @@ def add_edit_argument(parser):
     )
 
 
-def request_missing_metadata(bundle_subclass, args, initial_metadata=None):
+def request_missing_metadata(bundle_subclass, args, initial_metadata=None, bypass_editor= False):
```

Comment:
As I mentioned in the codalab repo, we should split this function into two:
- create_default_metadata: does everything before the test for editing
- request_missing_metadata: only does what the function name claims :-)

On Sun, Oct 11, 2015 at 11:16 PM, Stephen Koo <notifications@github.com>
wrote:

> In codalab/lib/metadata_util.py
> <https://github.com/codalab/codalab-cli/pull/166#discussion_r41726111>:
>
> > @@ -60,7 +60,7 @@ def add_edit_argument(parser):
> >      )
> >
> >
> > -def request_missing_metadata(bundle_subclass, args, initial_metadata=None):
> > +def request_missing_metadata(bundle_subclass, args, initial_metadata=None, bypass_editor= False):
>
> It might also help to add a brief comment about how bypass_editor should
> affect behavior here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/166/files#r41726111>.
>